.pre-commit-config.yaml

```diff
@@ -8,6 +8,7 @@ repos:
         args: [--style_pkg=styler, --style_fun=tidyverse_style]
     -   id: lintr
     -   id: readme-rmd-rendered
+        exclude: reports/README.md
```

Comment:
Necessary to prevent this check from running, since it thinks this README is coming from a `REAMDE.Rmd` file.